torch/utils/data/dataloader.py

```diff
@@ -72,7 +73,7 @@ def default_collate(batch):
         return torch.LongTensor(batch)
     elif isinstance(batch[0], float):
         return torch.DoubleTensor(batch)
-    elif isinstance(batch[0], str):
```

Comment:
`str` and `unicode` inherit from `basestring` so having that as the only argument in the tuple should be enough, right?
```
Python 2.7.10 (default, Jul 30 2016, 19:40:32) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> issubclass(str, basestring)
True
>>> issubclass(unicode, basestring)
True
```